src/quickfix.c

```diff
@@ -2335,6 +2347,7 @@ copy_loclist_entries(qf_list_T *from_qfl, qf_list_T *to_qfl)
 		    from_qfp->qf_pattern,
```

Comment:
The new flag has to be copied to the new location list in copy_loclist().  Also, do you need to copy the per quickfix item user_data in copy_loclist_entries()?

Reply:
I think you also need to copy the per-item user_data in the copy_loclist_entries() function?